AlphaWallet/Market/UniversalLinkHandler.swift

```diff
@@ -127,8 +126,8 @@ public class UniversalLinkHandler {
         let expiry = Array(bytes[16...19])
         let contractAddress = Array(bytes[20...39])
         let v = String(bytes[104], radix: 16)
-        let r = Data(bytes: Array(bytes[72...103])).hex()
-        let s = Data(bytes: Array(bytes[40...71])).hex()
+        let r = Data(bytes: Array(bytes[40...71])).hex()
+        let s = Data(bytes: Array(bytes[72...103])).hex()
```

Comment:
Is v, r, s packed the same way as normal and spawnable MagicLinks now? Maybe we can reuse `getVRSFromLinkBytes()` like the other 2 type of links, maybe passing in an offset?)